### PR TITLE
Update for linkerd 1.2.0

### PR DESCRIPTION
**Pull request by siggy**
In addition to linkerd version bump, primary change includes modifying
linkerd and namerd admin servers to serve on 0.0.0.0, rather than the
new default loopback.

### CODE REVIEW

File: k8s-daemonset/k8s/namerd.yml [hunk 1]
```diff
@@ -14,7 +14,8 @@ metadata:
 data:
   config.yml: |-
     admin:
-      port: 9990
+      ip: 0.0.0.0
+      port: 9991
```
Comment (esbie) [inline k8s-daemonset/k8s/namerd.yml]: https://buoyant.io/2016/11/04/a-service-mesh-for-kubernetes-part-iv-continuous-deployment-via-traffic-shifting/ has the admin port as 9990, any particular reason for the change here?

Comment (siggy) [reply k8s-daemonset/k8s/namerd.yml]: oof, i was trying to be consistent with (default) namerd admin ports, and differentiate them from linkerd admin ports. i'm tempted to update the blog to match this (also realizing i'll need to update the readme in this subdirectory to match this change). what do you think?

Comment (esbie) [reply k8s-daemonset/k8s/namerd.yml]: works for me! whichever way you want to go is 👍

Comment (siggy) [reply k8s-daemonset/k8s/namerd.yml]: updated https://buoyant.io/2016/11/04/a-service-mesh-for-kubernetes-part-iv-continuous-deployment-via-traffic-shifting/ to 9991

**Pull request merged**